### PR TITLE
update the bet disk

### PR DESCRIPTION
**Pull request by feilin-flexcompute**

### CODE REVIEW

File: docs/source/examples/betDiskSteady_caseStudy.rst [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+
+.. note::
+
+   The only difference between the above 5 case configuration files is the :code:`BETDisks/twists` distribution.
+
+For detailed instructions to upload a mesh, run a case and download the results, please refer to the :ref:`Quick Start <quickstart>` section of this documentation - these details will not be covered in this case study.
+
+Some tips on setting the input quantities related with BET can be found at :ref:`bet_input`. One thing needs to be noted that in the high fidelity DES of XV-15 rotor blades, there is no hub, so to match the model in high fidelity simulations, the chord length in :math:`r<0.09R` should be 0 in BET simulations. Therefore, in the case configuration file, the chord length is set to 0 right before the first cross section (:math:`r=0.09R`). This setting leads to the radial distribution similar to "chords_distribution_1" shown in :ref:`bet_input`. The forces and moments related to the BET Disk is saved at "bet_forces_v2.csv". A detailed description on this file can be found at :ref:`betDiskLoadingNote`. Here we will convert the non-dimensional values in the above csv file to dimensional values:
```
[the remark this answers is not in the document]
Comment (feilin-flexcompute) [reply docs/source/examples/betDiskSteady_caseStudy.rst]: fixed.

File: docs/source/examples/betDiskSteady_caseStudy.rst [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+
+In helicopter hovering mode, there is no freestream velocity. Five blade collective angles are considered in the current study: :math:`0^o, 3^o, 5^o, 10^o, 13^o` at :math:`r/R=0.75`, corresponding to low, medium and high disk loadings. The flow conditions are:
+
+* Tip Mach Number, defined as :math:`U_\text{tip}/C_{\infty}`, is 0.69, so :math:`U_\text{tip}=0.69C_\infty`.
+* Reynolds Number (based on reference chord (14 inch) and blade tip speed) = :math:`4.95\times 10^6`.
+* Reference Temperature = 288.15 K.
+
+Here are some points to setup the case configuration file:
+
+1. :code:`freestream/Mach` is set to 0, because there is no freestream velocity in hovering mode.
```
[the remark this answers is not in the document]
Comment (feilin-flexcompute) [reply docs/source/examples/betDiskSteady_caseStudy.rst]: yes, I tried, but somehow the omega is renumbered from 1. Maybe something in between break the numbering accumulator

File: docs/source/examples/betDiskSteady_caseStudy.rst [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+.. caution::
+   
+   In some simulations, the freestream is not treated to be in sea level standard on purpose. For example, in the "case 1a" from `3rd AIAA CFD High Lift Prediction Workshop <https://hiliftpw.larc.nasa.gov/Workshop3/testcases.html>`_, the viscosity of the freestream is adjusted to analyze the full-scale geometry at wind tunnel conditions, so please set the freestream properities based on your requirements. 
+
+.. _BET_XV15_hovering_caseStudy:
+
+Helicopter Hovering Mode
+---------------------------
+
+In helicopter hovering mode, there is no freestream velocity. Five blade collective angles are considered in the current study: :math:`0^o, 3^o, 5^o, 10^o, 13^o` at :math:`r/R=0.75`, corresponding to low, medium and high disk loadings. The flow conditions are:
```
[the remark this answers is not in the document]
Comment (feilin-flexcompute) [reply docs/source/examples/betDiskSteady_caseStudy.rst]: fixed

File: docs/source/examples/betDiskSteady_caseStudy.rst [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+   :scale: 80%
+   :align: center
+
+   XV-15 rotor blade's twist and chord radial distribution
+
+More information about the geometry and the high-fidelity detached eddy simulation using Flow360 can be found at 
+
+`Feilin Jia, John Moore and Qiqi Wang, 2022 Assessment of Detached Eddy Simulation and Sliding Mesh Interface in Predicting Tiltrotor Performance in Helicopter and Airplane Modes <https://arxiv.org/pdf/2201.11560.pdf>`_
+
+In this case study, the steady Navier-Stokes solver coupled with the Blade Element Theory Disk method is used to simulation the XV-15 rotor blades under two flight conditions: helicopter hovering mode and airplane mode. Compared to the high fidelity unsteady simulations with full rotor geometry in the above paper, the mesh of BET Disk solver is much coarser. The mesh used in the current case study contains 592K nodes. A overview of a slice of the mesh is shown below
```
[the remark this answers is not in the document]
Comment (feilin-flexcompute) [reply docs/source/examples/betDiskSteady_caseStudy.rst]: fixed

File: docs/source/examples/betDiskSteady_caseStudy.rst [hunk 5]
```diff
@@ -0,0 +1,359 @@
...
+      :width: 49%
+
+   .. |mesh4| image:: figures_BET_Tutorial/mesh_592K_zoom4.png
+      :width: 49%
+   
+   .. rst-class:: center1
+
+   Overview of volume mesh used in BET Disk simulations of XV-15 rotor, successively zooming in from top left->top right->bottom left->bottom right. 
+
+Before the simulation sections, the freestream quantities are shown below because these quantites are useful to setup the non-dimensional variables in case configuration and translate the non-dimensional output variabled into dimensional values. More information on non-dimensionalization in Flow360 can be found at :ref:`nondimensionalization_Flow360`. The grid unit :math:`L_{gridUnit}=1\,\text{inch}=0.0254\,\text{meter}` in the above mesh. In this case study, the freestream is in sea level standard, shown in :numref:`tab_air_isa`
```
[the remark this answers is not in the document]
Comment (feilin-flexcompute) [reply docs/source/examples/betDiskSteady_caseStudy.rst]: fixed, thanks

File: docs/source/capabilities/bladeElementTheory.rst [hunk 6]
```diff
@@ -41,27 +193,31 @@ The non-dimensional moment is defined as
...
 
-where :math:`r` is the distance between the node to the axis of rotation. :math:`\text{chord}_\text{ref}` is the dimensional refererence chord length. :math:`R` is the radius of the rotor disk. 
+where :math:`r` is the dimensional distance between the node to the axis of rotation. :math:`\text{chord}_\text{ref}` is the dimensional refererence chord length. :math:`R` is the dimensional radius of the rotor disk. 
 
-.. note::
+.. important::
 
-   All the quantities in the right hand side of :eq:`defBETForce`, :eq:`defBETMoment`, :eq:`defBETCt` and :eq:`defBETCq` are **dimensional**, which are different from the **non-dimensional** values in :ref:`betDisksInputParameters` of Flow360.json.
+   All the quantities in the right hand side of :eq:`defBETForce`, :eq:`defBETMoment`, :eq:`defBETCt` and :eq:`defBETCq` are **dimensional**, which are different from the **non-dimensional** values in :ref:`betDisksInputParameters` of Flow360.json. For example, at the first disk's first blade's first radial location :math:`r=\text{Disk0_Blade0_R0_Radius}\times L_\text{gridUnit}`. The conventions for non-dimensionalization in Flow360 can be found at :ref:`nondimensionalization_Flow360`.
```
[the remark this answers is not in the document]
Comment (feilin-flexcompute) [reply docs/source/capabilities/bladeElementTheory.rst:220]: fixed.

File: docs/source/capabilities/bladeElementTheory.rst [hunk 7]
```diff
@@ -16,7 +16,159 @@ Based on Blade Element Theory, Flow360 provides 2 related solvers, which can be
...
+BET input
+-----------
+
+.. note::
+   
+   In the BETDisks section, all input quantities are non-dimensional, except the :code:`twists` and :code:`alphas` (both in degree). The convention for non-dimensionalization in Flow360 can be found at :ref:`nondimensionalization_Flow360`. 
+
+Some input parameters related with BET solver in Flow360 are explained:
+
+1. **radius**. Any grid points enclosed by the cylinder defined by "radius", "center" and "axisOfRotation" will have aerodynamic forces imposed according to blade element theory.
```
[the remark this answers is not in the document]
Comment (feilin-flexcompute) [reply docs/source/capabilities/bladeElementTheory.rst]: fixed

File: docs/source/conventions/conventions.rst [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+Besides the above non-dimensional quantities, there are also many **coefficients** commonly used in the community of computational fluid dynamics, e.g. pressure coefficient (:math:`C_p`), skin friction coefficient (:math:`C_f`), lift coefficient (:math:`C_L`), drag coefficient (:math:`C_D`), etc. Flow360 also exports the above coefficients found in :ref:`volumeOutput <volumeOutputInputParameters>`, :ref:`surfaceOutput <surfaceOutputInputParameters>`, :ref:`sliceOutput <sliceOutputInputParameters>` as well as the "Forces" tab of web interface. 
+
+.. caution::
+
+   It should be noted that the reference velocity :math:`U_\text{ref}` used to calculate the :math:`C_p, C_f, C_D, C_L` can be set via "freestream/Mach" or "freestream/MachRef" by users. Its definition can be found in :ref:`case configuration <Flow360json>`. It is not the same as the reference velocity (:math:`C_\infty`) for non-dimensionalization in :numref:`tab_nondimensionalization_flow360`.
+
+It should also be noted that the "freestream/Reynolds" is based on the given reference velocity :math:`U_\text{ref}` and :math:`L_\text{gridUnit}`.
+
```
[the remark this answers is not in the document]
Comment (feilin-flexcompute) [reply docs/source/conventions/conventions.rst:35]: added the FAQ sections for non-dimensionalization

File: docs/source/conventions/nondimensionalization_flow360.csv [hunk 9]
```diff
@@ -0,0 +1,12 @@
+Property@ Reference value for non-dimen.@ Examples in Flow360
+Length@ :math:`L_\text{gridUnit}`@ geometry->{omentCenter, momentLength}, actuatorDisks->thickness, BETDisks->{radius, chordRef, thickness, bladeLineChord, chords, sectionalRadiuses}, {volumeOutput, surfaceOutput}->wallDistance
```
[the remark this answers is not in the document]
Comment (feilin-flexcompute) [reply docs/source/conventions/nondimensionalization_flow360.csv]: fixed

File: docs/source/conventions/conventions.rst [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+===================================
+
+In Flow360, most input and output variables are non-dimensional. The non-dimensionalization reduces the number of free parameters and helps to provide better understanding of the underlying physics. A non-dimensional variable is obtained by dividing its dimensional counterpart by an appropriately selected constant like :eq:`def_nondim`
+
+.. math::
+   :label: def_nondim
+
+   \text{non-dimensional variable} = \frac{\text{dimensional variable}}{\text{reference value}}
+
+Theoretically speaking, the reference values for non-dimensionalization can be arbitrary as long as the resulting equations are identical to the original ones, but in practice, the reference values are usually selected based on some typical parameters of problems and flow chracteristics to avoid confusion. The following list shows some commonly used non-dimensional variables in both input configuration files and output files:
```
[the remark this answers is not in the document]
Comment (feilin-flexcompute) [reply docs/source/conventions/conventions.rst]: fixed, good catch

**Review by rundaji — APPROVED** [review]

**Pull request merged**